charts/newrelic-logging/values.yaml

```diff
@@ -239,3 +239,12 @@ daemonSet:
 # into the plugin.source attribute.
 # Can be set as a global: global.lowDataMode
 # lowDataMode: false
+
+extraVolumes: []
+# - name: systemdlog
+#   hostPath:
+#     path: /run/log/journal
+
+extraVolumeMounts: []
+# - name: systemdlog
+#   mountPath: /run/log/journal
```

Comment:
Please also add a tree for Windows:
```
windows:
    extraVolumes: []
    extraVolumeMounts: []
```

In the future, we will move all the windows-related values here (such as the `windowsNodeSelector`, `windowsOsList`, `enableWindows`...).